ui/app/templates/jobs/run/templates/index.hbs

```diff
@@ -1,10 +1,26 @@
 <section class="section">
-    <h1 class="title with-flex">Hey buddy!</h1>
-    <span>Here are some preset template we create for you to get started. You can also create custom templates for you or your team to use in the future.</span>
-    <main data-test-template-list>PLACEHOLDER FOR RADIO CARD GROUP</main>
-    {{!-- Add Conditional Logic when no templates available --}}
-    <div>
-        <button data-test-apply>PLACEHOLDER USE HDS</button>
-        <button data-test-cancel>PLACEHOLDER USE HDS</button>
-    </div>
+    <header class="run-job-header">
+        <h1 class="title is-3">Choose a template</h1>
+        <p>A plan will be requested before the job is submitted.  You can pick a template or write your own job spec.</p>
+    </header>
+    <main class="radio-group" data-test-template-list>
+        <Hds::Form::RadioCard::Group @alignment="left" as |G|>
```

Comment:
Seems to be a bug with this radio card group implementation: I'm able to select multiple: 
<img width="770" alt="image" src="https://user-images.githubusercontent.com/713991/211350159-e3d596c4-6ca4-484a-b236-1b6a3264b294.png">

Reply:
Patched.